docs/content_management/pages/create_custom_page_block.md

```diff
@@ -132,5 +132,8 @@ The following UDW configuration is used so only an `event` typed content item ca
 
 For more information, see [UDW configuration](browser.md#udw-configuration).
 
-Your custom page block is now ready. 
-Before you can use it in Page Builder, you must [enable it in Page field settings]([[= user_doc =]]/content_management/configure_ct_field_settings/#block-display).
+Your custom page block is now ready.
```

Comment:
```suggestion
Your custom page block is now registered in the system.
```

Similarly as below - I'd expand on what "ready" means (because it doesn't mean "ready to be used in Page Builder")